docs/environment.yml

```diff
@@ -35,4 +35,7 @@ dependencies:
   - pytest-cov
   - pytest-html
   - pytest-xdist
+  # language packs
+  - jupyterlab-language-pack-fr-FR
+  - jupyterlab_server >=2.8.1,<3
```

Comment:
Not sure why `jupyter_server` is not listed here to handle the example files on the demo site.

Maybe the example files are copied and indexed differently, need to check.

Reply:
Ah it's probably the chain of `myst-nb` -> `nbdime` -> `jupyter_server` like last time.

So we should probably list the dependency explicitly here so it's easier to know where it's coming from.